edge_computing/day_2_core_cnf_clusters/updating/update-cnf-update-prep.adoc

```diff
@@ -1,12 +1,12 @@
 :_mod-docs-content-type: ASSEMBLY
-[id="telco-update-cnf-update-prep"]
-= Configuring CNF pods before updating the telco core CNF cluster
+[id="update-cnf-update-prep"]
+= Configuring application pods before updating your OpenShift cluster
 include::_attributes/common-attributes.adoc[]
-:context: telco-update-cnf-update-prep
+:context: update-cnf-update-prep
 
 toc::[]
 
-Follow the guidance in link:https://redhat-best-practices-for-k8s.github.io/guide/[Red Hat best practices for Kubernetes] when developing cloud-native network functions (CNFs) to ensure that the cluster can schedule pods during an update.
+It is very important to have a standard set of guidelines for your applications that run in cloud-native environments. In the telecommunications industry most CNF (containerized network function) vendors follow the guidance in link:https://redhat-best-practices-for-k8s.github.io/guide/[Red Hat best practices for Kubernetes] to ensure that the cluster can schedule pods properly during an upgrade.
```

Comment:
Consider re-writing this some something that describes what's in this section, and just keep the best practices guide as an additional resource.
Something like:
````
Configure application pods to ensure workload availability during OCP updates. For example, use deployment strategies, pod disruption budgets, anti-affinity rules, and health probes to maintain high availability and prevent service disruption.
````